### DISCUSSION TITLE
Name of enum to represent the date format pattern name.

### DISCUSSION THREAD

**Discussion by jmiranda** [openboxes/openboxes#4693] [Polls]
In OBPIH-6520 we're proposing to make a change to how we render dates in GSPs to remove format and replace with nothing (case of default format) or to pass the formatName as an argument to each call. 

```
<g:formatDate date="${someDate}" formatName="${DateFormatEnumName.DEFAULT}/>
<g:formatDate date="${someDate}" formatName="${DateFormatEnumName.EXPIRY_DATE}/>
<g:formatDate date="${someDate}" formatName="${DateFormatEnumName.DEFAULT_DATETIME}/>
<g:formatDate date="${someDate}" formatName="${DateFormatEnumName.DEFAULT_TIME}/>
```

> Look up format from the default MessageSource / ResourceBundle (i18n/*.properties file) with this key. If format and formatName are empty, format is looked up with 'default.date.format' key. If the key is missing, 'yyyy-MM-dd HH:mm:ss z' formatting pattern is used.

Source: https://gsp.grails.org/latest/api/org/grails/plugins/web/taglib/FormatTagLib.html

**Comment by crossterra-agent** [comment]
- **Status**: DONE
- **Role**: Orchestrator
- **Step**: 
- **Link**: https://github.com/CrossTerra/monorepo/pull/1785#issuecomment-3971088004
- **Summary**: Developer addressed amcguiga review comment but pre-commit is failing; directed developer to fix and confirm before reviewer is requested.
